_includes/components/section-menu.njk

```diff
@@ -25,15 +25,17 @@
                 {% set hiddenAttribute = "hidden" %}
             {% endif %}
 
-            {% if (not page.anchors) or (page.anchors.length === 0) %}
+            {% set subpages = collections.all | eleventyNavigation(page.key) | excludeDrafts | sortByWeight %}
+            
+            {% if subpages.length > 0 %}
                 <ul {{ hiddenAttribute }}>
-                    {% set subpages = collections.all | eleventyNavigation(page.key) | sortByWeight %}
                     {{ menu(subpages) }}
                 </ul>
             {% else %}
+                {% set anchors = page.anchors or (page.htmlContent | toc) %}
```

Comment:
I'm sorry, maybe I wasn't clear enough on that, but the `anchors` property from the front matter should now be completely ignored in favor of the TOC automatically generated from the headings. Maintaining an explicit TOC in the front matter is simply too much work and very error prone.

Reply:
I'm happy to create a separate PR that removes the existing anchors and confirms the TOC works as expected.

Reply:
Sure! I guess it can't hurt to have the option to provide an explicit TOC - it's just that everything in `anchors` is like either wrong or outdated. While we're at it, let's change the name of the property to `toc` though.